File: src/commands/VolumeCommand.js
const Command = require('../structures/Command')

class VolumeCommand extends Command {
  constructor () {
    super('volume')
    this.name = 'Volume'
    this.category = 'music'
    this.aliases = ['vl', 'vol']
    this.usage = 'volume'
  }

  async run (message, args, server, { t }) {
    if (!args[0]) return this.explain(message)
    if (!this.client.lavalinkManager.manager.players.has(message.guild.id)) return message.reply(t('commands:music.notPlaying'))
    if (isNaN(parseInt(args[0]))) return message.reply(t('commands:music.nan'))
    if (parseInt(args[0]) >= 151) return message.reply(t('commands:music.maxVolume'))

    await this.client.player.get(message.guild.id).player.volume(parseInt(args[0]))
    await message.reply(t('commands:music.volChanged', { volume: parseInt(args[0]) }))
  }
}

module.exports = VolumeCommand

File: src/main/kotlin/com/mystery/meteora/client/commands/VolumeCommand.kt
package com.mystery.meteora.client.commands

import com.mystery.meteora.client.lavaPlayer.PlayerController
import com.mystery.meteora.controller.Config
import com.mystery.meteora.controller.Helper
import com.mystery.meteora.handler.annotations.Command
import com.mystery.meteora.handler.annotations.Description
import com.mystery.meteora.handler.annotations.Module
import com.mystery.meteora.handler.annotations.Usage
import com.mystery.meteora.handler.modules.BaseModule
import net.dv8tion.jda.api.EmbedBuilder
import net.dv8tion.jda.api.events.message.MessageReceivedEvent
import java.awt.Color

@Module("Volume", "music")
@Description("volume.description")
@Usage("volume.usage")

class VolumeCommand(ctx: MessageReceivedEvent, args: String, prefix: String, config: Config) : BaseModule(ctx, args, prefix, config) {
  @Command("volume", "vol")
  fun volume() {
    if (args == "") {
      Helper().explain(context, "volume", "Volume", prefix, config!!)
      return
    }
    val guildPlayer = PlayerController.findManager(context.guild.idLong)
    var string = ""
    if (guildPlayer == null) {
      context.channel.sendMessage("There isn't an active player in this server.").queue()
    } else {
      try {
        val playerVolume = PlayerController(context).manager.player.volume
        val volume = args.toInt()
        if (volume in 0..150) {
          if (volume == 0) string = "🔇 – Player muted"
          if (volume > playerVolume && volume != 0) string = "🔊 – Turning the volume up to `${volume}/150`"
          if (volume < playerVolume && volume != 0) string = "🔉 – Turning the volume down to `${volume}/150`"
          if (volume == playerVolume) string = "The volume is the same – the command didn't have effect"
          val embed = EmbedBuilder()
            .setDescription(string)
            .setColor(Color(204, 214, 221))
          PlayerController(context).manager.player.volume = args.toInt()
          context.channel.sendMessage(embed.build()).queue()
        } else {
          context.channel.sendMessage("The volume value _**must**_ be between 0 and 150.").queue()
        }
      } catch (error: NumberFormatException) {
        context.channel.sendMessage("Oops! Seems like your input is not a number.").queue()
      }
    }
  }
}